### PR TITLE
Fixes to get builds to work again

### PR DESCRIPTION
**Pull request by tomerk**
@shivaram @etrain

### CODE REVIEW

**Comment by etrain** [comment]
Did you run the full suite of tests locally on this? In particular - something which serializes `Image` would be good?

File: src/main/scala/utils/images/Image.scala [hunk 1]
```diff
@@ -282,7 +282,7 @@ trait VectorizedImage extends Image {
 
   def vectorToImageCoords(v: Int): Coordinate
```
Comment (shivaram) [inline src/main/scala/utils/images/Image.scala]: Can we make this method and the `coord` object `private` ? Right now `coord` is only used by this method, so it gives us some assurance it won't be accessed outside of it.

Comment (etrain) [inline src/main/scala/utils/images/Image.scala]: Yeah, I think that's a fine idea. We haven't done this much yet, but given
that this thing is potentially "dangerous" I guess we should go ahead and
do this.

**Comment by shivaram** [comment]
Did the tests pass locally ?

**Comment by shivaram** [comment]
Alright - since you confirmed this offline, I'm merging this now.

**Pull request merged**